web/public/js/controllers/user-profile-controller.js

```diff
@@ -27,6 +27,7 @@ function cdUserProfileCtrl($scope, $rootScope, $state, $window, auth, cdUsersSer
   $scope.editMode = false;
   $scope.publicMode = false;
   $scope.publicChampion = false;
+  $scope.loggedInUserIsMemberOfDojoChampion = false;
```

Comment:
Nope, you want to check the opposite, if the loggedInUser is a member of the profile's dojo he's viewing